Functions/Import/xASL_imp_DCM2NII_ConvertScan.m

```diff
@@ -77,16 +79,21 @@
         fprintf('Subject = %s, visit = %s, session = %s, scan = %s\n',scanFields.subjectID, scanFields.visitID, printSession, scan_name);
     end
 
-    % Defaults
-    bOneScanIsEnough = false;
-    bPutInSessionFolder = true; 
+%     % Defaults
+    bOneScanIsEnough = false; % PM: THIS USED TO BE TRUE FOR ANATOMICAL SCANS
+    bPutInSessionFolder = true; % THIS USED TO BE FALSE FOR ANATOMICAL SCANS
     switch scan_name
         case {'ASL4D', 'M0', 'ASL4D_RevPE', 'func_bold'}
             bPutInSessionFolder = true;
-        case {'T1', 'WMH_SEGM', 'FLAIR', 'T2', 'T1c'}
-            bPutInSessionFolder = false;
+            bAnatomical = false;        
```

Comment:
it would be good to initialize this also upfront. not only inside branches

Reply:
I actually think nothing should be initialized upfront; those defaults give problems.
Better to provide a default at the end, when an unknown `scan_name` is detected, with a warning. Initializing dependent variables upfront is asking for bugs, unless you assume that we always carefully think of every possible scenario when coding, which is not really possible in our convoluted code anymore ;)